README.rst

```diff
@@ -7,15 +7,15 @@ Local install
 Getting a local Trac install running is a bit tricky. Here are a few tricks
 that can help:
 
-* Install the requirements in the same virtualenv as djangoproject.com.
-* Follow the installation instructions in djangoproject/README.rst (especially
-  the database creation).
-* Use ``psql -U code.djangoproject -d code.djangoproject -c "INSERT INTO permission (username, action) VALUES ('anonymous', 'TRAC_ADMIN')"``
-  to give all permissions to the anonymous user.
-* Use the command ``tracd --port 9000 -s trac-env`` to serve Trac locally.
-* If you've modified the ``trackhack.scss`` file, use
-  ``sassc trac-env/htdocs/css/trachacks.scss trac-env/htdocs/css/trachacks.css -s compressed``
-  to compile it to CSS.
+1. Install the requirements in the same virtualenv as djangoproject.com.
```

Comment:
This list is more of a random collections of tips so I think it can remain an unordered list.